yapapi/golem.py

```diff
@@ -125,7 +142,7 @@ def __init__(
         if not strategy:
             strategy = self._initialize_default_strategy()
 
-        self._engine_args = {
+        self._engine_args: _EngineKwargs = {
```

Comment:
Attribute is `_engine_args`, type is `_EngineKwargs` - this hurts :)
I think we should rename the attribute.

Reply:
for me it's not an issue, since "keyword arguments" are still "arguments" ... but sure, will comply with your sense of order :)